src/bootstrap/build/sanity.rs

```diff
@@ -79,7 +79,7 @@ pub fn check(build: &mut Build) {
         }
 
         // Make sure musl-root is valid if specified
-        if target.contains("musl") {
+        if target.contains("musl") && target.contains("x86_64") {
```

Comment:
OK, I'll interpret that as: "let's add i686 here after we confirm we can cross compile std to it using rustbuild".